CONTRIBUTING.md

```diff
@@ -0,0 +1,39 @@
+# 💝 Code Style Guidelines
+
+## Principles
```

Comment:
Should we add something about tests here?

Reply:
We should definitely add more than just code style there.

Reply:
A good point of reference is the Notion document we have about code style. Maybe we should discuss it after the standup tomorrow and gather ideas? This'll definitely develop over time though.